glide-core/src/client/value_conversion.rs

```diff
@@ -241,6 +241,14 @@ pub(crate) fn expected_type_for_cmd(cmd: &Cmd) -> Option<ExpectedReturnType> {
 mod tests {
     use super::*;
 
+    #[test]
```

Comment:
I don't thing we need to add a test to this kind of conversions, take a look at the rest of the tests, they are for more "complex" conversions

Reply:
Are there test coverage requirements or plans for that?

Reply:
not that I know of, but what I believe is that we test new return types that we add (like map of string to double) or when we change commands return type only if it contains some flag

Reply:
Removed